docs/source/reference/kubernetes/kubernetes-getting-started.rst

```diff
@@ -296,6 +296,11 @@ FAQs
                 volumeMounts:       # Custom volume mounts for the pod
                   - mountPath: /foo
                     name: example-volume
+                resources:          # Custom resource requests and limits
+                  requests:
+                    rdma/rdma_shared_device_a: 1
+                  limits:
+                    rdma/rdma_shared_device_a: 1
```

Comment:
Is this relates to the `show-gpus`?

Reply:
No, just adding docs on how to add custom resources through pod_config. Could have been a separate PR :)